src/Symfony/Component/Security/Core/Tests/Authentication/Token/AbstractTokenTest.php

```diff
@@ -212,53 +209,59 @@ public function testSetUserSetsAuthenticatedToFalseWhenUserChanges($firstUser, $
     }
 
     public function getUserChanges()
+    {
+        $user = $this->getMockBuilder('Symfony\Component\Security\Core\User\UserInterface')->getMock();
```

Comment:
I'm just wondering since I'm not too familiar with the Symfony testing guidelines: Wouldn't it be easier and more refactoring-friendly to use the `::class` constants here instead of hand-writing the FQCN?

Reply:
Consistency in this case. I personally would update everything to use `::class`, but that might cause merge conflicts for no reason when merging changes from lower branches upwards.

Reply:
Fair enough :) Thanks!